admin_guide/node_problem_detector.adoc

```diff
@@ -0,0 +1,376 @@
+                        "reason": "AUFSUmountHung",
+                        "pattern": "task umount\\.aufs:\\w+ blocked for more than \\w+ seconds\\."
+                },
+                {
+                        "type": "permanent",
+                        "condition": "KernelDeadlock",
+                        "reason": "DockerHung",
+                        "pattern": "task docker:\\w+ blocked for more than \\w+ seconds\\."
+                }
+        ]
+    }
+----
+
+. Remove or add any NodeConditions (conditions) or events (rules) as needed. 
+
+. Restart running pods to apply the changes. To restart pods, you can delete all existing pods:
++
+[source,bash]
+----
+# oc delete pods -n openshift-node-problem-detector -l name=node-problem-detector
+----
+
+[NOTE]
+====
+The add-on manager does not support configuration maps. If you are running the node problem detector running as cluster add-on, 
+configuration overwriting is not supported.
+====
+
+[[admin-guide-node-problem-detector-customize]]
+== Configuring Custom Logging and Monitoring Tools
+
+You can use xref:admin-guide-node-problem-detector-fbk[file-based kernel logging], or 
+xref:admin-guide-node-problem-detector-monitor[custom monitoring tools]. 
+
+To enable plugins, edit the node problem detector daemonset:
+
+. Edit the Node Problem Detector configuration map with a text editor.
+[source,bash]
+----
+oc edit -n openshift-node-problem-detector configmap node-problem-detector
+----
+
+. Add the following flag to enable kernel-monitor, and docker-monitor.json:
+
+[source,yaml]
+----
+ docker-monitor.json: |
+    {
+        "plugin": "journald",""
+----
+
+<1> Sends the output to standard output (stdout).
+<2> Comma-separated path to the plugin configuration files.
+
+To display node problem detector output to standard output (stdout) and standard error (stderr) edit the node problem detector daemonset:
+
+[source,bash]
+----
+# oc edit ds node-problem-detector
+----
+
+. Add the following flag to set the output and enable kernel-monitor, and docker-monitor.jason:
+
+[source,yaml]
+----
+spec:
+  containers:
+  - command:
+    - node-problem-detector
+    - --alsologtostderr=true <1>
+      --log_dir="/tmp" <2>
+    - --system-log-monitors=/etc/npd/kernel-monitor.json,/etc/npd/docker-monitor.json <3>
+----
+
+<1> Sends the output to standard output (stdout).
+<2> Path to the error log.
+<2> Comma-separated path to the plugin configuration files.
+
+
+[[admin-guide-node-problem-detector-fbk]]
+=== Using File-Based Kernel Logging 
+
+The node problem detector supports file-based kernel logging. 
+
+You can use other file-based logging tools, such as link:https://github.com/kubernetes/node-problem-detector/tree/master/pkg/systemlogmonitor#system-log-monitor[System Log Monitor].
+Use the following command with a comma-separated list of paths to system log monitor configuration files:
+
+[source,bash]
+----
+# oc node-problem-detector --system-log-monitors <path to config files>
```

Comment:
This looks like a typo.  The `oc` should be removed and we should probably add a note that the `node-problem-detector` command line can be modified in the daemonset to add the `--system-log-monitors` switch. Basically, the method to configure this option is like the above one where the `oc edit ds` is used to modify the container's command. In fact, perhaps this section could be combined with the one above it since they're covering the configuration of the same thing.